es6/index.js

```diff
@@ -0,0 +1,21 @@
+import got from 'got-promise';
+
+function pluckValue(response) {
+  return response.body[0].Value;
```

Comment:
just that I wanted to treat it like a simple key/value store and ignore everything else

- [ ] return entire consul response